### PR TITLE
Farm 13 home update

### PR DESCRIPTION
**Pull request by AntoineCAU**
Styling with love on the Home component

### CODE REVIEW

**Review by Erriep — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -52,6 +53,7 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "husky": "4.3.8",
     "lint-staged": "^10.5.4",
+    "nodemon": "^2.0.7",
```
Comment (LoicBrassart) [inline package.json]: Naaaaaaaaaaaaaaaaan! >_<

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Nopenope

**Review by Erriep — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
NmE!

**Review by Erriep — DISMISSED** [review]

**Review by Erriep — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**